# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# Filename: package/.../keepalived/keepalived-1.1.13-types_h.patch
# Copyright (C) 2007 The OpenSDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- keepalived-1.1.13/keepalived/vrrp/vrrp_arp.c.orig	2007-08-06 22:57:11.000000000 +0200
+++ keepalived-1.1.13/keepalived/vrrp/vrrp_arp.c	2007-08-06 22:55:40.000000000 +0200
@@ -22,14 +22,14 @@
  * Copyright (C) 2001-2006 Alexandre Cassen, <acassen@linux-vs.org>
  */
 
-/* system includes */
-#include <linux/if_packet.h>
-
 /* local includes */
 #include "vrrp_arp.h"
 #include "memory.h"
 #include "utils.h"
 
+/* system includes */
+#include <linux/if_packet.h>
+
 /* global vars */
 char *garp_buffer;
 int garp_fd;
